docs/examples/promise/04_listen_to_multiple_balances_change/index.js

```diff
@@ -22,7 +22,7 @@ async function main () {
 
   // Subscribe and listen to several balance changes
   api.query.genericAsset.freeBalance.multi([[CENNZ, ALICE], [CENNZ, BOB]], (balances) => {
-    console.log('Change detected, new balances: ', balances);
+    console.log('Change detected, new balances: ', balances.toString());
```

Comment:
Change detected, new balances:  [ <BN: 33b2e3c9fd0803ce7ff6f55>, <BN: 33b2e3c9fd0803ce80090ab> ]
Change detected, new balances:  [ <BN: 33b2e3c9fd0803ce7ff3f1c>, <BN: 33b2e3c9fd0803ce800c0e4> ]

changed the appearance to

Change detected, new balances:  999999999999999999999950620,1000000000000000000000049380
Change detected, new balances:  999999999999999999999938275,1000000000000000000000061725